portal-web/test/functional/com/liferay/portalweb/macros/GDPR.macro

```diff
@@ -72,6 +72,20 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
+	macro anonymizeAllStagedDatainApplicationCP {
```

Comment:
Name the macro to anonymizeStagedDatainApplicationCP since this is not actually for delete all staged entries.